js/index.js

```diff
@@ -8,35 +8,103 @@ const siteContent = {
   },
+  footer: {
+    copyright: "Copyright Great Idea! 2018"
+  }
 };
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.src = siteContent["nav"]["img-src"];
+
+/////////////navItems//////////////
+let navItems = document.querySelectorAll("nav a");
+navItems.forEach((el, i) => {
+  el.textContent = `${siteContent.nav[`nav-item-${[i + 1]}`]}`;
+  el.style.color = "green";
+});
+////////////////////H1 Text////////////////////
+let h1text = document.querySelector(".cta .cta-text h1");
+siteContent.cta.h1 = siteContent.cta.h1.replace(/\s/g, "\n");
+h1text.innerText = siteContent.cta.h1;
+
+/////////////Button////////////////////
+let btn = document.querySelector("button");
+btn.innerText = siteContent.cta.button;
+
+///////////CTA Image////////////////////
+let ctaImg = document.querySelector("#cta-img");
+ctaImg.src = `${siteContent.cta["img-src"]}`;
+
+////////////////H4 Titles///////////////
+let h4Titles = document.querySelectorAll("h4");
+h4Titles[0].innerText = siteContent["main-content"]["features-h4"];
+h4Titles[1].innerText = siteContent["main-content"]["about-h4"];
+h4Titles[2].innerText = siteContent["main-content"]["services-h4"];
+h4Titles[3].innerText = siteContent["main-content"]["product-h4"];
+h4Titles[4].innerText = siteContent["main-content"]["vision-h4"];
+h4Titles[5].innerText = siteContent.contact["contact-h4"];
+
+/////////////////////Middle Image/////////////////////
+let midImg = document.querySelector("#middle-img");
+midImg.src = `${siteContent["main-content"]["middle-img-src"]}`;
+
+////////////////////Paragraphs/////////////////////
+let paras = document.querySelectorAll("p");
+paras[0].innerText = siteContent["main-content"]["features-content"];
+paras[1].innerText = siteContent["main-content"]["about-content"];
+paras[2].innerText = siteContent["main-content"]["services-content"];
+paras[3].innerText = siteContent["main-content"]["product-content"];
+paras[4].innerText = siteContent["main-content"]["vision-content"];
+//////////////////Contact//////////////////////////
+siteContent.contact.address = siteContent.contact.address.replace(
+  /t S/,
+  "t\n S"
+);
+paras[5].innerText = siteContent.contact.address;
+paras[6].innerText = siteContent.contact.phone;
+paras[7].innerText = siteContent.contact.email;
+/////////////////////////////Footer/////////////////
+paras[8].innerText = siteContent.footer.copyright;
+
+////////////////New Elements///////////////////
+let navList = document.querySelector("nav");
+let newEl1text = document.createTextNode("Sign In");
+let newEl1 = document.createElement("a");
+newEl1.appendChild(newEl1text);
+let newEl2text = document.createTextNode("Find Friends");
+let newEl2 = document.createElement("a");
+newEl2.appendChild(newEl2text);
+navList.prepend(newEl1);
+navList.appendChild(newEl2);
+navList.style.color = "green";
+navItems[0].createAttribute("href='#'");
```

Comment:
```suggestion
navItems = document.querySelectorAll("nav a");
navItems[0].setAttribute("href", "#");
navItems[7].setAttribute("href", "#");
navItems[0].style.color = "green";
navItems[7].style.color = "green";
```
So your list of navItems does not contain the new elements you added, you need to get those elements again now that you have created new ones. So in the first line I added there, we re-assign the value of navItems to the new set of links you have. Then we can accurately target that array. I would use setAttribute rather than createAttribute in this situation.